Updates README with brief project description (#13)
googleinterns/univiz: README.md

# step192-2020

## Project Overview

> The UniViz project aims to create an interactive visualization dashboard
> to allow students to best organize their college application portfolio,
> from colleges they are applying to to application fees to financial aid
> considerations. Stay tuned for more updates!

Reviewer: Some rewording:
```suggestion
The UniViz project aims to create an interactive visualization dashboard
to allow students organize their college application portfolio, based on
colleges they are applying to. You can also track application fees,
deadlines, and financial aid considerations. Stay tuned for more updates!
```
